pkg/install/templates/cache-svc.yaml.tmpl

```diff
@@ -0,0 +1,31 @@
+{{- if or (eq .CacheBackend "memcached") (eq .CacheBackend "") -}}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: memcached
+{{- if .Namespace }}
+  namespace: {{ .Namespace }}
+{{- end }}
+spec:
+  ports:
+    - name: memcached
+      port: 11211
+  selector:
+    name: memcached
+{{- else if eq .CacheBackend "redis" -}}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis
```

Comment:
Wouldn't it be better if this corresponded with the default value for the service name? That way there is less to accidentally get wrong.

Reply:
(notwithstanding I advise above to remove this from `fluxctl install` entirely)

Reply:
keep in mind that removing from `fluxctl install` will require additional time since I'll have to find a better solution on how to hack the _e2e_ tests to evaluate also the Redis server.